src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -92,7 +91,7 @@ public function __construct()
     public function getPasses()
     {
         return array_merge(
-            array($this->mergePass),
+            array(new RegisterEnvVarProcessorsPass(), $this->mergePass),
```

Comment:
now runs even earlier.. anything to do? Patched PhpDumper test for now.

Reply:
LGTM

Reply:
Wait, I just realized that this may just make it impossible to register a new env processor in practice, isn't it?